src/helpers/momentHelper.js

```diff
@@ -74,3 +74,56 @@ export const isActive = (item) => {
 
   return true;
 };
+
+/**
+ *
+ * Find the item with the closest date range to today.
+ *
+ * @param {DataItem[]} data an array of objects, each with a `dates` array
+ *
+ * @return {DataItem | null} the item with the closest date range to today, or null if no such item exists
```

Comment:
do your prefer `null` over `undefined` in general? what is the benefit of having `null`s here for `findClosestItem`?

Reply:
I don't think there is any advantage or disadvantage of using null over undefined, but I believe that undefined would be more accurate. You are right.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/909/commits/d664c3f0a762f98b5502f2da02380b3c1cbd2e5b